lib/connections/limiter.go

```diff
@@ -124,6 +127,87 @@ func (lim *limiter) VerifyConfiguration(from, to config.Configuration) error {
 	return nil
 }
 
+func nextChange(to config.Configuration) time.Duration {
+	now := time.Now()
+
+	beginHour := to.Options.ScheduledRatesFromHour
+	beginMinute := to.Options.ScheduledRatesFromMinute
+	begin := time.Date(now.Year(), now.Month(), now.Day(), beginHour, beginMinute, 0, 0, now.Location())
+
+	tillBegin := begin.Sub(now)
+
+	var waitTime time.Duration
+
+	if tillBegin >= 0 {
+		waitTime = tillBegin
+	} else {
+		endHour := to.Options.ScheduledRatesToHour
```

Comment:
The usual style is to use early returns, e.g. here:

```suggestion
	if tillBegin >= 0 {
		return tillBegin
	}
	endHour := to.Options.ScheduledRatesToHour
```

Reply:
Done